includes/payments/class-give-payment.php

```diff
@@ -948,9 +948,10 @@ public function save() {
 
 					case 'date':
 						$args = array(
-							'ID'        => $this->ID,
-							'post_date' => date( 'Y-m-d H:i:s', strtotime( $this->date ) ),
-							'edit_date' => true,
+							'ID'            => $this->ID,
+							'post_date'     => date( 'Y-m-d H:i:s', strtotime( $this->date ) ),
+							'post_date_gmt' => date( 'Y-m-d H:i:s', strtotime( $this->date ) ),
```

Comment:
@nishitlangaliya will local date and `gmt` date equal for any region?

Reply:
@ravinderk , As per our discussion on call I have made the changes and update the post_date_gmt with gmt version of post_date

Thanks